src/test/json.test.ts

```diff
@@ -99,15 +101,15 @@ function assertVisit(input: string, expected: VisitorCallback[], expectedErrors:
 	}, {
 		disallowComments
 	});
-	assert.deepEqual(errors, expectedErrors);
-	assert.deepEqual(actuals, expected, JSON.stringify(actuals));
+	assert.deepStrictEqual(errors, expectedErrors);
+	assert.deepStrictEqual(actuals, expected, JSON.stringify(actuals));
 }
 
 function assertNodeAtLocation(input: Node | undefined, segments: Segment[], expected: any) {
 	let actual = input && findNodeAtLocation(input, segments);
 	assert.deepEqual(actual ? getNodeValue(actual) : void 0, expected);
```

Comment:
Have not replaced this with `assert.deepStrictEqual` because then the test fails.